capture/parsers/tls.c

```diff
@@ -874,12 +884,24 @@ void moloch_parser_init()
         MOLOCH_FIELD_TYPE_STR_GHASH,  MOLOCH_FIELD_FLAG_CNT,
         (char *)NULL);
 
+    ja3StrField = moloch_field_define("tls","lotermfield",
```

Comment:
Can you move the 2 field creations inside an if with config.ja3Strings

Reply:
sure; commited.